pkg/runner/runner.go

```diff
@@ -135,6 +139,23 @@ func finalExecutionResult(output []byte, err error) (result testkube.ExecutionRe
 	return result
 }
 
+// areChecksSuccessful verifies the summary at the end of the execution to see if any of the checks failed
+func areChecksSuccessful(summary string) bool {
```

Comment:
Maybe we can add some regular expressions to this as it seems to me that it may cause strange behavior in the future

Reply:
Good catch Nico! Indeed we could have some false positives here, 
Let's maybe merge this one and open new task for integrating junit output formatter 

https://github.com/Mattihew/k6-to-junit

We could also save output as `junit` and allow it to pass for later use 

It would be first junit powered executor.

Reply:
I also saw there's a way to get a better output - that would help us refactor other functions that iterate over the lines as well. I will create a follow-up ticket.